### PR TITLE
change to meta to possibly localise oauth dialog boxes per T166472

### PR DESCRIPTION
**Pull request by Pix1234**

### CODE REVIEW

**Comment by Pix1234** [comment]
Per bryan davis this should do what we want

File: config/00-main.yaml [hunk 1]
```diff
@@ -14,5 +14,5 @@ database:
   dbname: wikilabels
 
 oauth:
-  mw_uri: https://www.mediawiki.org/w/index.php
+  mw_uri: https://www.meta.wikimedia.org/w/index.php
```
Comment (Ladsgroup) [inline config/00-main.yaml]: Besides that I'm not sure if we should do this at all, www.meta.wikimedia.org is wrong, you need to omit www.

**Review by Ladsgroup — CHANGES_REQUESTED** [review]

**Review by adamwight — CHANGES_REQUESTED** [review]
We should continue discussion on the task since I don't see agreement to go in the direction taken here. 

Also, small bookkeeping thing so I don't have to use Phabricator search again ;-) : please reference the bug from your commit messages, something like:

Bug: https://phabricator.wikimedia.org/T166472

Thanks!

**Comment by Pix1234** [comment]
@adamwight  I meant to let me fix this

**Pull request merged**